tools/protodoc/requirements.txt

```diff
@@ -1 +1,12 @@
-PyYAML==5.3.1
+PyYAML==5.3.1 \
```

Comment:
?

Reply:
Reverting, this was an experiment around https://github.com/envoyproxy/envoy/issues/12951